src/main/holochainManager.ts

```diff
@@ -60,22 +60,23 @@ export class HolochainManager {
     lairUrl: string,
     bootstrapUrl: string,
     signalingUrl: string,
+    iceUrls?: string[],
     rustLog?: string,
     wasmLog?: string
   ): Promise<HolochainManager> {
     const adminPort = process.env.ADMIN_PORT
       ? parseInt(process.env.ADMIN_PORT, 10)
       : await getPort();
 
-    const conductorConfig = rustUtils.defaultConductorConfig(
+    const conductorConfig = defaultConductorConfig(
       adminPort,
       rootDir,
       lairUrl,
       bootstrapUrl,
       signalingUrl,
-      "*",
+      "moss-admin-main",
```

Comment:
Is this intentional? I wouldn't expect Moss references here

Reply:
obviously not intentional, no

Reply:
Also not seeing a change here